server/index.js

```diff
@@ -14,42 +14,72 @@ const port = 9000;
 app.use(cors());
 app.use(bodyParser.json());
 app.use(express.static('client/dist'));
+app.use('/:id', express.static('client/dist'));
 
-app.get('/api/products', (req, res) => {
-  controller.ikea_products.get(req, res, (err, data) => {
+
+// app.get('/api/products', (req, res) => {
+//   controller.ikea_products.get(req, res, (err, data) => {
+//     if (err) {
+//       // eslint-disable-next-line no-console
+//       console.error('error connecting to server');
+//       return;
+//     }
+//     // eslint-disable-next-line no-console
+//     console.log('ikea_products connected');
+//     res.send(data).status(200);
+//   });
+// });
+
+app.get('/api/products/', (req, res) => {
```

Comment:
Do you need an endpoint that will retrieve every product from your database? This will take forever.